src/index.js

```diff
@@ -1,28 +1,21 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { BrowserRouter as Router } from "react-router-dom";
-import './index.css';
-import firebaseConfig from "./config/FirebaseConfig"
-// import App from './App';
-import * as serviceWorker from './serviceWorker';
-// import withFirebaseAuth from 'react-with-firebase-auth';
-import * as firebase from "firebase/app";
+import React from 'react'
+import ReactDOM from 'react-dom'
+import './index.css'
+import { firebaseConfig } from './config/FirebaseConfig'
+import { App } from './App'
+import * as serviceWorker from './serviceWorker'
+import * as firebase from 'firebase/app'
 import 'firebase/analytics'
-import 'firebase/auth';
+import 'firebase/auth'
 import 'semantic-ui-css/semantic.min.css'
-import Dtty from './components/Dtty';
-
 
 // Initialize Firebase
-firebase.initializeApp(firebaseConfig);
-firebase.analytics();
+firebase.initializeApp(firebaseConfig)
+firebase.analytics()
 
-ReactDOM.render(<Router>
-    <Dtty />
-</Router>
-    , document.getElementById('root'));
+ReactDOM.render(<App />, document.getElementById('root'))
```

Comment:
This is how you should virtually always instantiate your React application. Use `<App />` here instead of what you did.